tests/_utils/start_tidb_cluster_impl

```diff
@@ -125,6 +125,8 @@ if [[ "$tidb_config" != "" ]]; then
 else
     cat - >"$OUT_DIR/tidb-config.toml" <<EOF
 split-table = true
+alter-primary-key = true
+new_collations_enabled_on_first_bootstrap = true
```

Comment:
Shall we also enable new ci collation in `start_tls_tidb_cluster_impl`

Reply:
updated